### ISSUE TITLE
VTT subtitles wrong position after adjusting font-size

### ISSUE THREAD

**Issue by hugochief** [videojs/video.js#7942]
### Description

I'm using Video JS to load VTT subtitles. I have a subtitle on top using "line:1" on the VTT file and another on the bottom using "line:95%".

Everything works fine when font size is set at 100%. However, when I try increasing the font size to 200% the subtitle on top grows in the wrong direction, growing towards the top edge of the video instead of growing towards the centre.

![Doc1](https://user-images.githubusercontent.com/83137908/193067838-0fc9125f-0e22-473b-bd35-f0a3185e2b83.png)

I've tried editing this part of the script file:

``if (overrides.fontPercent && overrides.fontPercent !== 1) {
      var fontSize = window.parseFloat(cueDiv.style.fontSize);
      cueDiv.style.fontSize = fontSize * overrides.fontPercent + 'px';
      cueDiv.style.height = 'auto';
      cueDiv.style.top = 'auto';`

By deleting the two last lines, the issue on the top subtitle was solved, but it made the bottom one have the same issue, by having it grow towards the bottom of the video instead of towards the centre.

![MEg6N](https://user-images.githubusercontent.com/83137908/193069209-0b49d777-0e5a-4f51-a2b2-1d9537657f3b.png)

### Reduced test case

https://codepen.io/hugo-sousa-the-bashful/pen/MWGQVdy

### Steps to reproduce

1.Create VTT subtitle with top and bottom lines
2.Increase font size on VideoJS player
3.

### What version of Video.js are you using?

7.21.0

### What browser(s) including version(s) does this occur with?

Chrome 106

### What OS(es) and version(s) does this occur with?

Windows 10

**Comment by gkatsev** [comment]
Yeah, this is a bit tricky because these caption settings are happening at a different layer than it should be. It should be happening inside vttjs, which renders the captions, but currently it's happening at the "app" level (i.e. Video.js). Due to this, it isn't simple to fix things since, as you mentioned, making one change affects the other.
Adding support for these inside of vttjs is the proper method, but it's not a quick change, unfortunately.

**Comment by lyriclearn** [comment]
Thanks for the response @gkatsev.
Maybe in a future update then.
